paasta_tools/contrib/create_paasta_playground.py

```diff
@@ -38,5 +50,56 @@ def main():
     )
 
 
+def run_local_zookeeper(user, zookeeper_port):
```

Comment:
```suggestion
def run_local_zookeeper(user: str, zookeeper_port: int) -> None:
```

Reply:
additionally: maybe this could be `ensure_running_local_zookeeper`) so that we don't have a `create_` function that actually runs something :p